rmgpy/rmg/main.py

```diff
@@ -1260,26 +1270,38 @@ def make_seed_mech(self, first_time=False):
                     q += 1
                 self.name = name + str(q)
 
+    def make_seed_mech(self):
+        """
+        causes RMG to make a seed mechanism out of the current chem_annotated.inp and species_dictionary.txt
+        this seed mechanism is outputted in a seed folder within the run directory and automatically
+        added to as the (or replaces the current) 'Seed' thermo and kinetics libraries in database
+
+        `initialize_seed_mech` should be called once before this function is ever called.
+
+        This also writes the filter tensors to the `filters` sub-folder for restarting an RMG job from a seed mechanism
```

Comment:
Could you reword this docstring? It's written a bit weirdly and isn't really accurate.

I think it should start with "This method saves a seed mechanism..." or "Save a seed mechanism...". It is also not generating it from the chem_annotated.inp and species_dictionary.txt but rather the actual reaction model.

Reply:
Updated! I should have updated this the last time I modified this function in a PR